cmd/podman/stats.go

```diff
@@ -122,6 +117,9 @@ func statsCmd(c *cliconfig.StatsValues) error {
 		}
 	} else if all {
 		containerFunc = runtime.GetAllContainers
+	} else {
+		// default to get all *running* containers for Docker compat
+		containerFunc = runtime.GetRunningContainers
```

Comment:
as reported by golangci-lint, this chunk should not be needed since containerFunc already defaults to `containerFunc := runtime.GetRunningContainers`